core/tests/utils_fixtures.py

```diff
@@ -141,17 +140,18 @@ def create_group(name, rwd=None, rw=None, ru=None, ro=None):
     return group
 
 
-def create_bailleur():
-    return Bailleur.objects.create(
-        nom="3F",
-        siret="12345678901234",
-        capital_social="123000.50",
-        ville="Marseille",
-        signataire_nom="Patrick Patoulachi",
-        signataire_fonction="PDG",
-        signataire_date_deliberation=datetime.date(2014, 10, 9),
-        signataire_bloc_signature="Mon PDG",
-    )
+def create_bailleur(**kwargs):
+    d = {
+        "nom": "3F",
+        "siret": "12345678901234",
+        "capital_social": "123000.50",
+        "ville": "Marseille",
+        "signataire_nom": "Patrick Patoulachi",
+        "signataire_fonction": "PDG",
+        "signataire_date_deliberation": datetime.date(2014, 10, 9),
+        "signataire_bloc_signature": "Mon PDG",
+    } | kwargs
+    return Bailleur.objects.create(**d)
```

Comment:
Note pour plus tard: on pourrait gagner en souplesse dans les tests and utilisant [Factoryboy](https://factoryboy.readthedocs.io/en/stable/index.html) pour gérer ces fixtures.

Ex:
```python
class BailleurFactory(factory.django.DjangoModelFactory):
    class Meta:
        model = Bailleur

    nom = Sequence(lambda n: f"Bailleur {n}")
    siret = FuzzyText(length=14, chars=string.digits)
    ...

class ProgrammeFactory(factory.django.DjangoModelFactory):
    class Meta:
        model = Programme

    bailleur = factory.SubFactory(BailleurFactory)
    ...

programme = ProgrammeFactory()
programme = ProgrammeFactory(bailleur__siret="11111111111111")
...
```